examples/src/main/scala/examples/hybrid/HLocalInterface.scala

```diff
@@ -41,7 +41,7 @@ class HLocalInterface(override val viewHolderRef: ActorRef,
   override protected def onSemanticallySuccessfulModification(mod: HybridBlock): Unit = {
     if (!block) {
       mod match {
-        case wb: PowBlock =>
+        case _: PowBlock =>
           posForgerRef ! StartForging
           powMinerRef ! MineBlock
```

Comment:
Maybe this comment is not relevant to this PR, but one thing that I find confusing here is that an expected symmetry seems to be broken. 

When we receive a PosBlock, we stop forging and start mining. I was expecting that we would stop mining and start forging when we receive a PowBlock. But, instead, what we see here is a command for the `powMinerRef` to mine a block instead of stop mining.

Could you explain this, perhaps as a comment in the code, if you consider appropriate?